lib/vanity/adapters/mock_adapter.rb

```diff
@@ -137,9 +137,11 @@ def ab_add_participant(experiment, alternative, identity)
         alt[:participants] << identity
       end
 
-      def ab_seen(experiment, identity, alternative)
-        if ab_assigned(experiment, identity) == alternative.id
-          alternative
+      def ab_seen(*args)
```

Comment:
Ah, good point - I did it this way because I wanted to avoid shadowing the method parameters with the block parameters, but I hadn't considered the docs. I'll sort this out, thanks. 👍